server/routes/partyRoute.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express';
+import PartyController from '../controllers/PartyController';
+import PartyValidator from '../middlewares/PartyValidator';
+
+const router = express.Router();
+
+router.post(
+  '/parties',
+  PartyValidator.validateFields,
+  PartyController.postParty,
+);
+
+router.get(
+  '/parties/:id',
+  PartyController.getParty,
```

Comment:
You're not validating that I actually passed in an ID. I think you should do that.